### PR TITLE
docs: update images for master and v17

### PR DESCRIPTION
**Pull request by mmcallister**
Logo updates tracked in https://github.com/gravitational/teleport/issues/46300.

### CODE REVIEW

File: docs/img/use-teleport/connect-my-computer-status@2x.png
Comment (ravicious) [inline docs/img/use-teleport/connect-my-computer-status@2x.png:1]: The new image shows Connect My Computer in Discover, not in Teleport Connect.

I think we can keep the Connect My Computer screenshots as they are.

File: docs/img/use-teleport/connect-my-computer-top-bar@2x.png
Comment (ravicious) [inline docs/img/use-teleport/connect-my-computer-top-bar@2x.png:1]: Same issue as above.

File: docs/pages/connect-your-client/teleport-connect.mdx
Comment (ravicious) [inline docs/pages/connect-your-client/teleport-connect.mdx:292]: If we are going to restore previous versions of Connect My Computer images, can we bring back this one as well?

File: docs/img/access-controls/access-lists/create-new-access-list.png
Comment (zmb3) [inline docs/img/access-controls/access-lists/create-new-access-list.png:1]: There's a lot of plain white pixels in this image. What do you think about making the window more narrow before grabbing the screenshot?

File: docs/img/access-controls/dual-authz/teleport-4-bob-request.png
Comment (zmb3) [inline docs/img/access-controls/dual-authz/teleport-4-bob-request.png:1]: This doesn't look like Teleport 17. The left nav here looks like a very old version of Teleport.

Did you grab this screenshot from a real instance or did you just paste the new logo over the old one?

File: docs/img/access-controls/dual-authz/teleport-6-ivan-approve.png
Comment (zmb3) [inline docs/img/access-controls/dual-authz/teleport-6-ivan-approve.png:1]: Similarly, this also has the old Teleport 12-style nav.

File: docs/img/access-controls/dual-authz/teleport-7-bob-assume.png
Comment (zmb3) [inline docs/img/access-controls/dual-authz/teleport-7-bob-assume.png:1]: Similarly, this also has the old Teleport 12-style nav.

File: docs/img/access-requests/approved-request.png
Comment (zmb3) [inline docs/img/access-requests/approved-request.png:1]: Old nav here too.

File: docs/img/access-requests/new-role-request.png
Comment (zmb3) [inline docs/img/access-requests/new-role-request.png:1]: Old nav here too.

File: docs/img/access-requests/submit-request.png
Comment (zmb3) [inline docs/img/access-requests/submit-request.png:1]: Old nav here too.

File: docs/img/architecture/tls-routing-connection-upgrade.png
Comment (zmb3) [inline docs/img/architecture/tls-routing-connection-upgrade.png:1]: Old "T" logo still present on the right side in this image.

File: docs/img/desktop-access/passwordless-desktop.png
Comment (zmb3) [inline docs/img/desktop-access/passwordless-desktop.png:1]: Old nav here too.

**Review by zmb3 — CHANGES_REQUESTED** [review]
I didn't finish the review, but most of the screenshots here are super outdated and don't look anything like Teleport 17.

It's more work, but I think this is a good opportunity to get fresh screenshots so that the docs are an accurate representation of the product.

File: docs/img/access-controls/access-lists/create-new-access-list.png
Comment (mmcallister) [reply docs/img/access-controls/access-lists/create-new-access-list.png:1]: This has been cut down

File: docs/img/access-controls/dual-authz/teleport-4-bob-request.png
Comment (mmcallister) [reply docs/img/access-controls/dual-authz/teleport-4-bob-request.png:1]: Thank you, updated! These were for the other branch.

File: docs/img/access-controls/dual-authz/teleport-6-ivan-approve.png
Comment (mmcallister) [reply docs/img/access-controls/dual-authz/teleport-6-ivan-approve.png:1]: Updated

File: docs/img/access-controls/dual-authz/teleport-7-bob-assume.png
Comment (mmcallister) [reply docs/img/access-controls/dual-authz/teleport-7-bob-assume.png:1]: Updated

File: docs/img/architecture/tls-routing-connection-upgrade.png
Comment (mmcallister) [reply docs/img/architecture/tls-routing-connection-upgrade.png:1]: Fixed!

File: docs/img/use-teleport/connect-my-computer-status@2x.png
Comment (mmcallister) [reply docs/img/use-teleport/connect-my-computer-status@2x.png:1]: Restored

File: docs/pages/connect-your-client/teleport-connect.mdx
Comment (mmcallister) [reply docs/pages/connect-your-client/teleport-connect.mdx:292]: Done!

File: docs/img/access-requests/approved-request.png
Comment (mmcallister) [reply docs/img/access-requests/approved-request.png:1]: Updated

File: docs/img/access-requests/submit-request.png
Comment (mmcallister) [reply docs/img/access-requests/submit-request.png:1]: Updated

File: docs/img/desktop-access/passwordless-desktop.png
Comment (mmcallister) [reply docs/img/desktop-access/passwordless-desktop.png:1]: Updated

File: docs/img/access-controls/device-trust/hosted-jamf.png
Comment (zmb3) [inline docs/img/access-controls/device-trust/hosted-jamf.png:1]: I'd make this window more narrow. 2300 pixels wide and more than half of it is empty space.

File: docs/img/access-controls/dual-authz/accessRequested-new.png
Comment (zmb3) [inline docs/img/access-controls/dual-authz/accessRequested-new.png:1]: Similarly, I'd make this more narrow.

Also, minor nit, but it looks like we generally use kebab-case for naming these images, but you've mixed camelCase and kebab-case here.

File: docs/img/access-controls/dual-authz/role-new-request.png
Comment (zmb3) [inline docs/img/access-controls/dual-authz/role-new-request.png:1]: Similar comments on a lot of these images, they're very wide and mostly empty space.

File: docs/img/webui_billing_cycle.png
Comment (zmb3) [inline docs/img/webui_billing_cycle.png:1]: I don't believe this is from Teleport 17, because we've removed the access management tab.

File: docs/pages/admin-guides/access-controls/guides/dual-authz.mdx
Comment (zmb3) [inline docs/pages/admin-guides/access-controls/guides/dual-authz.mdx:201]: Does this still look good in the rendered view? With the expanded role name it looks like the columns might not align anymore.

Comment (mmcallister) [reply docs/pages/admin-guides/access-controls/guides/dual-authz.mdx:201]: Recreated the table, I think this should fix it

File: docs/img/webui_billing_cycle.png
Comment (mmcallister) [reply docs/img/webui_billing_cycle.png:1]: Removed the image and related paragraph

File: docs/img/access-controls/dual-authz/role-new-request.png
Comment (mmcallister) [reply docs/img/access-controls/dual-authz/role-new-request.png:1]: Trimmed the image

File: docs/img/access-controls/dual-authz/accessRequested-new.png
Comment (mmcallister) [reply docs/img/access-controls/dual-authz/accessRequested-new.png:1]: This one wasn't used, deleted the image from the directory

File: docs/pages/connect-your-client/teleport-connect.mdx
Comment (ravicious) [reply docs/pages/connect-your-client/teleport-connect.mdx:292]: I see it's removed still. 🫥

File: docs/pages/admin-guides/access-controls/guides/dual-authz.mdx
Comment (mmcallister) [reply docs/pages/admin-guides/access-controls/guides/dual-authz.mdx]: Does this look ok:


tsh_req_list_output

File: docs/pages/connect-your-client/teleport-connect.mdx
Comment (ravicious) [reply docs/pages/connect-your-client/teleport-connect.mdx:292]: When I open the "Files changed" tab in this PR, it tells me that [`docs/img/use-teleport/connect-my-computer-setup@2x.png` has been removed](https://github.com/gravitational/teleport/pull/48824/files#diff-f74b32aafbd54623636487e75fba29e77023075a638ce303242a7076de5aa074) and that [it has been removed from `teleport-connect.mdx` as well](https://github.com/gravitational/teleport/pull/48824/files#diff-f9d1425ebeb5497eec68d311c2a20b5fe1616ea10fe943863cd80a452d2f75d0).

Comment (mmcallister) [reply docs/pages/connect-your-client/teleport-connect.mdx]: Apologies @ravicious , I was looking at the wrong 'connect-my-computer-x' image. Restored:

https://github.com/gravitational/teleport/pull/48824/commits/f935faca177ed78c66b0177b617db1fd8b4fa354

File: docs/img/review-request.png
Comment (zmb3) [inline docs/img/review-request.png:1]: This image still looks outdated.

File: docs/img/request-access.png
Comment (zmb3) [inline docs/img/request-access.png:1]: This image is still outdated. Teleport 17 doesn't have this side nav.

File: docs/img/login-success.png
Comment (zmb3) [inline docs/img/login-success.png:1]: There shouldn't be a white background on the Teleport logo like there is in this image. Was it captured from a real Teleport UI or manually patched?

File: docs/img/management/access-list-web-ui.png
Comment (zmb3) [inline docs/img/management/access-list-web-ui.png:1]: I see we switched from light to dark theme on some of these images. Does this mean that the page the image is used on might now alternate between light and dark screenshots?

File: docs/img/externalauditstorage/integration.png
Comment (zmb3) [inline docs/img/externalauditstorage/integration.png:1]: Why are we changing this one? The previous image already used the Teleport 17 nav and had the correct logo. (It also did a better job at highlighting the EAS tile)

File: docs/img/enterprise/license-warning.png
Comment (zmb3) [inline docs/img/enterprise/license-warning.png:1]: This image is outdated (both the left nav and the banner at the top no longer look like this).

Let me know if you need help getting an updated screenshot here. We can put this banner on a cloud tenant for you.

File: docs/img/enterprise/license-expired.png
Comment (zmb3) [inline docs/img/enterprise/license-expired.png:1]: (See comment on the next image - it applies here too)

File: docs/img/desktop-access/select-desktop.png
Comment (zmb3) [inline docs/img/desktop-access/select-desktop.png:1]: This is super outdated. Desktop Access was in preview for Teleport 8!

**Review by zmb3 — COMMENTED** [review]
We're getting closer but there's still a number of outdated images that we should update.

File: docs/img/desktop-access/select-desktop.png
Comment (mmcallister) [reply docs/img/desktop-access/select-desktop.png:1]: This one, `passwordless-desktop.png` wasn't used in any of the content, I will remove it.
The updated Desktop image is above,`desktop-access/passwordless-desktop.png`

File: docs/img/enterprise/license-warning.png
Comment (mmcallister) [reply docs/img/enterprise/license-warning.png:1]: That would be great, I don't think I can generate this one. Thank you!

File: docs/img/externalauditstorage/integration.png
Comment (mmcallister) [reply docs/img/externalauditstorage/integration.png:1]: Not sure why I created this one..  Restored the previous image
😣

File: docs/img/management/access-list-web-ui.png
Comment (mmcallister) [reply docs/img/management/access-list-web-ui.png:1]: For the page this is on, it's the only UI .png image. The rest are CLI examples in
`admin-guides/infrastructure-as-code/managing-resources/access-list.mdx
` but I can recreate a light version if preferred!

File: docs/img/request-access.png
Comment (mmcallister) [reply docs/img/request-access.png:1]: Replaced with the new one

File: docs/img/review-request.png
Comment (mmcallister) [reply docs/img/review-request.png:1]: Added the new one that was missed.

File: docs/img/access-requests/new-role-request.png
Comment (zmb3) [inline docs/img/access-requests/new-role-request.png:1]: This image is titled `new-role-request`, but the screenshot is showing a resource request, not a role request.

File: docs/img/access-requests/submit-request.png
Comment (zmb3) [inline docs/img/access-requests/submit-request.png:1]: WDYT about only screenshotting the right half of this image? We would waste less space with empty black pixels and there's less chance of the image going out of date if we focus only on the relevant part.

Comment (mmcallister) [reply docs/img/access-requests/submit-request.png:1]: Good call. Updated!

File: docs/img/desktop-access/session-recording@2x.png
Comment (zmb3) [inline docs/img/desktop-access/session-recording@2x.png:1]: This image has the old Teleport 15-style nav.

File: docs/img/management/check-users-web-ui.png
Comment (zmb3) [inline docs/img/management/check-users-web-ui.png:1]: Lots of empty space in this image. I wonder if a screenshot is even necessary here.

File: docs/img/request-access.png
Comment (zmb3) [inline docs/img/request-access.png:1]: Crop this one to get rid of all that empty space?

**Review by zmb3 — APPROVED** [review]
A few final comments, but nothing that needs to block approval.

Thanks so much for all the work on this one, I know it's a slog.

File: docs/img/login-success.png
Comment (mmcallister) [reply docs/img/login-success.png:1]: I edited this one; I tried to set up GitHub OAuth but I didn't have an Organization to use. Tbh this image seems unnecessary, I'll delete and edit the page accordingly.

File: docs/img/access-requests/new-role-request.png
Comment (mmcallister) [reply docs/img/access-requests/new-role-request.png:1]: Grabbed the wrong one - updated with the role request

File: docs/img/management/check-users-web-ui.png
Comment (mmcallister) [reply docs/img/management/check-users-web-ui.png:1]: Deleted this one and edited the page

File: docs/img/request-access.png
Comment (mmcallister) [reply docs/img/request-access.png:1]: Cropped

File: docs/img/desktop-access/session-recording@2x.png
Comment (mmcallister) [reply docs/img/desktop-access/session-recording@2x.png:1]: Updated

**Comment by mmcallister** [comment]
@ravicious would you mind giving this another look? Thank you!

File: docs/img/access-controls/access-lists/create-new-access-list.png
Comment (ravicious) [inline docs/img/access-controls/access-lists/create-new-access-list.png:1]: The text above this screenshot needs an update. The text was already out of date and with the nav change it's even more outdated. In the new nav, it's just Add New -> Access List.

https://github.com/gravitational/teleport/blob/bd8e52c133815afc0daa481ab664843456ba0015/docs/pages/admin-guides/access-controls/access-lists/guide.mdx#L48-L54

File: docs/img/access-controls/dual-authz/approveRequest-new.png
Comment (ravicious) [inline docs/img/access-controls/dual-authz/approveRequest-new.png:1]: This file doesn't appear to be used anywhere. I think it might be a copy of `docs/img/access-controls/dual-authz/approve-new-request.png`.

File: docs/img/access-requests/approved-request.png
Comment (ravicious) [inline docs/img/access-requests/approved-request.png:1]: This screenshot should show an approved request, not a pending request.

File: docs/img/architecture/proxy-peering@1.2x.png
Comment (ravicious) [inline docs/img/architecture/proxy-peering@1.2x.png:1]: Could you create a separate issue and work with the design team to update this image? Because of the gradient, the new logo looks tacked, which doesn't look great IMHO. At the same time I don't want this to block this PR.

Comment (ravicious) [reply docs/img/architecture/proxy-peering@1.2x.png:1]: Looking at other images in this folder, I think we might end up wanting to modify the original SVGs instead of replacing them with PNGs.

File: docs/img/architecture/tls-routing-connection-upgrade.png
Comment (ravicious) [inline docs/img/architecture/tls-routing-connection-upgrade.png:1]: This one especially needs an update from the original SVG, since the png looks super blurred on a high DPI screen.

File: docs/img/elevated-access-role.yaml
Comment (ravicious) [inline docs/img/elevated-access-role.yaml:1]: Unused file.

**Review by ravicious — COMMENTED** [review]
I reviewed changes up until `docs/img/enterprise/license.png`, will continue tomorrow.

File: docs/pages/admin-guides/infrastructure-as-code/managing-resources/user-and-role.mdx
Comment (ravicious) [inline docs/pages/admin-guides/infrastructure-as-code/managing-resources/user-and-role.mdx:368]: There's no management panel anymore. This should guide towards Access -> Users instead.

File: docs/img/management/datadog-diagram.png
Comment (ravicious) [inline docs/img/management/datadog-diagram.png:1]: Teleport event handler still technically uses the old logo.

File: docs/img/management/fluentd-diagram.png
Comment (ravicious) [inline docs/img/management/fluentd-diagram.png:1]: Same here wrt old logo for the event scheduler.

File: docs/img/request-access.png
Comment (ravicious) [inline docs/img/request-access.png:1]: The text around this says:

> Users can request access using the Web UI by visiting the "Access Requests" tab and clicking "New Request":

https://docs-otu9t2vtk-goteleport.vercel.app/docs/admin-guides/api/access-plugin/#step-55-test-your-plugin

There's no such tab anymore and the screenshot shows an already created request, not where to go to create a new one.

IMHO we could remove the screenshot and update `docs/pages/includes/plugins/create-request.mdx` to guide the user towards Identity -> Access Requests -> New Access Request instead.

**Review by ravicious — COMMENTED** [review]
Reviewed everything until `docs/img/request-access.png`, will continue tomorrow!

File: docs/img/access-controls/dual-authz/approveRequest-new.png
Comment (mmcallister) [reply docs/img/access-controls/dual-authz/approveRequest-new.png:1]: Removed

File: docs/img/access-controls/access-lists/create-new-access-list.png
Comment (mmcallister) [reply docs/img/access-controls/access-lists/create-new-access-list.png:1]: Updated verbiage

File: docs/img/management/datadog-diagram.png
Comment (mmcallister) [reply docs/img/management/datadog-diagram.png:1]: for this one, I didn't edit that part. we can update when that changes

File: docs/img/elevated-access-role.yaml
Comment (mmcallister) [reply docs/img/elevated-access-role.yaml:1]: Deleted

File: docs/img/architecture/proxy-peering@1.2x.png
Comment (mmcallister) [reply docs/img/architecture/proxy-peering@1.2x.png:1]: Ok, I'll create a seperate issue for them to update these

File: docs/img/k8s/mini-diagrams/k8s-to-teleport-mono.png
Comment (mmcallister) [reply docs/img/k8s/mini-diagrams/k8s-to-teleport-mono.png:1]: I'll add all of these diagrams into the new issue. These could definitely be prettier  =)

File: docs/img/request-access.png
Comment (mmcallister) [reply docs/img/request-access.png:1]: Changed the verbiage. I had a screengrab for this, updated to the correct one.

File: docs/pages/admin-guides/infrastructure-as-code/managing-resources/user-and-role.mdx
Comment (mmcallister) [reply docs/pages/admin-guides/infrastructure-as-code/managing-resources/user-and-role.mdx:368]: Changed

File: docs/img/access-requests/approved-request.png
Comment (mmcallister) [reply docs/img/access-requests/approved-request.png:1]: Removed. This page could use an overhaul, I'll open this as a separate issue.

File: docs/img/team-diagram.png
Comment (ravicious) [inline docs/img/team-diagram.png:1]: Unused image.

File: docs/img/trusted-clusters/trusted-clusters@1.5x.png
Comment (ravicious) [inline docs/img/trusted-clusters/trusted-clusters@1.5x.png:1]: `docs/pages/reference/architecture/trustedclusters.mdx` still uses `docs/img/architecture/trusted-clusters@1.5x.svg` which has old logos.
 
https://docs-jxt52qzvh-goteleport.vercel.app/docs/reference/architecture/trustedclusters/

File: docs/img/windows-desktop-admins.yaml
Comment (ravicious) [inline docs/img/windows-desktop-admins.yaml:1]: Unused file.

File: docs/img/workload-identity/intro-diagram.png
Comment (ravicious) [inline docs/img/workload-identity/intro-diagram.png:1]: For some reason the top logo is of much worse quality than the bottom one.

File: docs/pages/admin-guides/access-controls/sso/github-sso.mdx
Comment (ravicious) [inline docs/pages/admin-guides/access-controls/sso/github-sso.mdx:309]:
```suggestion
After logging in, you will receive a "Login Successful" window, which you can then close.
```

**Review by ravicious — APPROVED** [review]
LGTM once the remaining comments are addressed!

**Comment by mmcallister** [comment]
For references, here's the Design team's task for their part in updating the diagrams:

https://app.asana.com/0/1208760782936960/1208896236369924/f

Once done I will upload their changes in a separate PR.

File: docs/img/trusted-clusters/trusted-clusters@1.5x.png
Comment (mmcallister) [reply docs/img/trusted-clusters/trusted-clusters@1.5x.png:1]: This will be updated by the Design team

**Comment by ptgott** [comment]
@mmcallister Is there a v17 backport PR for this one? Thanks!

**Pull request merged**
